WalkingChallenge/AKFCausesService.swift

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright © 2017 Aga Khan Foundation
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * 3. The name of the author may not be used to endorse or promote products
+ *    derived from this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE AUTHOR "AS IS" AND ANY EXPRESS OR IMPLIED
+ * WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO
+ * EVENT SHALL THE AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+ * WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR
+ * OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+ * ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ **/
+
+enum AKFCausesEndPoint {
+  case healthcheck
+  case participant(String)
+  case participants
+  case team(Int)
+  case teams
+}
+
+extension AKFCausesEndPoint {
+  public var rawValue: String {
+    switch self {
+    case .healthcheck:
+      return ""
+    case .participant(let fbid):
+      return "participants/\(fbid)"
+    case .participants:
+      return "participants"
+    case .team(let id):
+      return "teams/\(id)"
+    case .teams:
+      return "teams"
+    }
+  }
+}
+
+class AKFCausesService: Service {
+  public static var shared: AKFCausesService =
+    AKFCausesService(baseURL: "https://akf-causes.subshell.org")
```

Comment:
I'd prefer if this was in some config (AppConfig?) file instead of hard coded here. That way if we have a prod/staging environment in the future we can keep all the config vars in one place.